ql/src/java/org/apache/hadoop/hive/ql/exec/spark/status/SparkJobMonitor.java

```diff
@@ -82,7 +82,7 @@
   protected SparkJobMonitor(HiveConf hiveConf) {
     monitorTimeoutInterval = hiveConf.getTimeVar(
         HiveConf.ConfVars.SPARK_JOB_MONITOR_TIMEOUT, TimeUnit.SECONDS);
-    inPlaceUpdate = InPlaceUpdates.inPlaceEligible(hiveConf);
+    inPlaceUpdate = InPlaceUpdate.canRenderInPlace(hiveConf) && !SessionState.getConsole().getIsSilent();
```

Comment:
reason being the  InPlaceUpdate now belongs to hive-common and SessionState to hive-exec/ ql, hence reverse dependency issue.